pkg/skaffold/build/jib/init.go

```diff
@@ -89,13 +89,30 @@ func validate(path string, enableGradleAnalysis bool) []ArtifactConfig {
 	var builderType PluginType
 	var executable, wrapper, taskName, searchString, consoleFlag string
 	switch {
-	case strings.HasSuffix(path, "pom.xml"):
+	case isPomFile(path):
 		builderType = JibMaven
 		executable = "mvn"
 		wrapper = "mvnw"
 		searchString = "<artifactId>jib-maven-plugin</artifactId>"
```

Comment:
TBH I think just searching for `jib-maven-plugin` is enough.  After all, this string would be fooled if someone had `jib-maven-plugin` as a `<dependency>`.